src/arc.ts

```diff
@@ -1,5 +1,5 @@
 import { NotificationStatus } from "reducers/notifications";
-import { getNetworkId, getNetworkName } from "./lib/util";
+import { getNetworkId, getNetworkName, targetedNetwork } from "./lib/util";
```

Comment:
why this change ?

Reply:
see the corresponding change and the comment where I explained, in lib/util.ts

Reply:
is that a must for supporting kovan ?

Reply:
Without that change we would be worsening the problem by adding a dependency to the circle. 
 There is less risk in fixing this than in not (which is easy to say because this change is virtually risk-free).